extensions.json

```diff
@@ -1481,6 +1481,13 @@
       "download": "https://github.com/tuxtina/json2yaml/releases/download/v0.2.0/json2yaml-0.2.0.vsix",
       "version": "0.2.0"
     },
+    {
+      "id": "Tyriar.sort-lines",
+      "repository": "https://github.com/Tyriar/vscode-sort-lines",
+      "version": "1.9.0",
```

Comment:
Nit: When pinning a specific `"version"`, please also pin a specific `"checkout"` (e.g. release branch, tag, or commit ID), otherwise the script will publish from the default branch. (Alternatively, you can simply remove the `"version"` pin.)

Reply:
I removed `"version"`